pkg/helm/handlers/handlers.go

```diff
@@ -100,3 +102,26 @@ func (h *helmHandlers) HandleHelmList(user *auth.User, w http.ResponseWriter, r
 	res, _ := json.Marshal(resp)
 	w.Write(res)
 }
+
+func (h *helmHandlers) HandleGetRelease(user *auth.User, w http.ResponseWriter, r *http.Request) {
+	queryParams := r.URL.Query()
+	ns := queryParams.Get("ns")
+	chartName := queryParams.Get("release_name")
+
+	conf := h.getActionConfigurations(h.ApiServerHost, ns, user.Token, &h.Transport)
+	release, err := h.getRelease(chartName, conf)
+	if err != nil {
+		serverutils.SendResponse(w, http.StatusBadRequest, serverutils.ApiError{Err: fmt.Sprintf("Failed to find helm release: %v", err)})
+		return
+	}
+
+	w.Header().Set("Content-Type", "application/json")
+
+	rawManifest, err := json.Marshal(release)
+
+	if err != nil {
+		serverutils.SendResponse(w, http.StatusBadRequest, serverutils.ApiError{Err: fmt.Sprintf("Failed to find helm release: %v", err)})
```

Comment:
I'd use 500 Internal Server Error since we don't expect this to fail.

We probably should add this to the other places that are missing `json.Marshal` error handling

Reply:
Done, will be raising another PR to add error handling to remaining `json.Marshal` places.